pkg/cli/newapp/newapp.go

```diff
@@ -375,8 +375,8 @@ func (o *AppOptions) RunNewApp() error {
 			"app.kubernetes.io/component": result.Name,
 		}
 
-		for _, name := range o.Config.ImageStreams {
-			if name != "" {
+		for _, is := range o.Config.ImageStreams {
+			if name, _, ok := imageutil.SplitImageStreamTag(is); ok && len(name) > 0 {
```

Comment:
@soltysh I don't think we need to check `ok` here. `latest` is an implicit tag name if not provided - we just need to strip out the `:` if it is present.

Reply:
@adambkaplan yeah, that's the problem I just fixed in the last update, ptal